### PR TITLE
separator fix

### PR DESCRIPTION
**Pull request by krystofex**
## Description

Change separator component class

## Related Issue(s)

x-overflow on /workspaces page

## Release Notes
```release-note
Fix x-overflow on the /workspaces page
```

### CODE REVIEW

**Comment by roboquat** [comment]
@krystofex: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **laushinka** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @laushinka` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by bigint** [comment]
Screenshot 2021-12-05 at 4 46 23 PM

Perfect I can verify x-overflow is fixed, great job 👏🏻

**Review by bigint — APPROVED** [review]
LGTM 🚀

**Comment by roboquat** [comment]
@bigint: changing LGTM is restricted to collaborators

<details>

In response to [this](https://github.com/gitpod-io/gitpod/pull/7065#pullrequestreview-823378486):

>LGTM 🚀 


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7065#pullrequestreview-823378486" title="Approved">bigint</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **laushinka** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @laushinka` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by laushinka** [comment]
Hi @krystofex, thanks for the contribution! Could you describe what the problem was before? Maybe in a screenshot? :)

**Comment by bigint** [comment]
@laushinka If you visit https://gitpod.io/workspaces you can see the x scrollbar overflow the screen

Screenshot 2021-12-06 at 6 00 56 PM

But this PR helps us to remove the buggy x-overflow

https://github.com/gitpod-io/gitpod/pull/7065#issuecomment-986210981

**Comment by laushinka** [comment]
Works as advertised! Thanks @krystofex :)

/lgtm

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: e6fec37551af11d2816700bc9e99c50f657c2627</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7065#pullrequestreview-823378486" title="Approved">bigint</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/7065#issuecomment-986796162" title="LGTM">laushinka</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [laushinka]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7065#pullrequestreview-823378486" title="Approved">bigint</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/7065#issuecomment-986836701" title="Approved">laushinka</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7065#issuecomment-986836701" title="Approved">laushinka</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [laushinka]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/dashboard/src/components/Separator.tsx
Comment (jankeromnes) [inline components/dashboard/src/components/Separator.tsx:8]: 😍 

Just seeing this now -- many thanks @krystofex @filiptronicek @bigint and @laushinka! 🎉 💯

**Pull request merged**
